client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/index.tsx

```diff
@@ -23,8 +25,20 @@ interface Props {
 
 export const Analyzer: FC< Props > = ( { onComplete, onSkip, hideImporterListLink = false } ) => {
 	const translate = useTranslate();
+	const isEnglishLocale = useIsEnglishLocale();
 	const [ siteURL, setSiteURL ] = useState< string >( '' );
 
+	// TODO: Remove extra steps for non-English locales once we have translations.
+	const oldSubtitle = translate( 'Drop your current site address below to get started.' );
+	const newSubtitle = translate(
+		"Drop your current site address below to get started. In the next step, we'll measure your site's performance and confirm its eligibility for migration."
+	);
+	const isTranslationAvailableForNewSubtitle = hasTranslation(
```

Comment:
I remember we shouldn't pass variables to the `translate()` function, is there a similar rule for `hasTranslation()`?

Reply:
I don't know for sure. But in this case I've repeated the string because even if we passed a variable to hasTranslation, we'd have to duplicate anyway

Reply:
Ah, makes sense. We don't want to pass the translation here.
Thanks for clarification!